plotly/tests/test_optional/test_grid/test_grid.py

```diff
@@ -0,0 +1,32 @@
+"""
+test_grid:
+==========
+
+A module intended for use with Nose.
+
+"""
+from __future__ import absolute_import
+
+from unittest import TestCase
+
+from plotly.exceptions import InputError
+from plotly.grid_objs import Grid
+
+import pandas as pd
+
+
+class Test_Dataframe_to_Grid(TestCase):
```

Comment:
Sorry, I can see why that's confusing ;__;.

I was talking about camelCase vs snake_case (hence the 🐍 and 🐫).